src/styles/_catalog.scss

```diff
@@ -1,7 +1,8 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  grid: auto-flow / repeat(auto-fit, minmax(195px, 1fr));
```

Comment:
```suggestion
  grid: auto-flow / repeat(auto-fit, minmax(195px, 1fr));
```
why 195px?

Reply:
Hello!
I thought that I need to take away 5px which include border-radius...and it came out 195)
Fixed all.